TestAssets/TestProjects/StandaloneApp/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc3-*"
+      }
+    }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "win8-x64": {},
```

Comment:
I'll do some additional testing. This unblocked me yesterday.

---
In reply to: [66106986](https://github.com/dotnet/core-setup/pull/97#discussion_r66106986) [](ancestors = 66106986)